content/docs/quickstart/go.md

```diff
@@ -3,98 +3,150 @@ title: Go Quick Start
 layout: quickstart
 short: Go
 description: This guide gets you started with gRPC in Go with a simple working example.
+protoc-version: 3.11.4
 ---
 
 ### Prerequisites
 
-- Go version 1.6 or higher.
-
-For installation instructions, see Go's [Getting Started](https://golang.org/doc/install) guide.
+- **Go** version 1.12+. For installation instructions, see Go's [Getting
+  Started](https://golang.org/doc/install) guide.
 
 #### gRPC
 
-Use the following command to install gRPC.
+Use the following command to install gRPC as a [module][]:
 
 ```sh
-$ go get -u google.golang.org/grpc
+$ export GO111MODULE=on # Enable module-aware mode
+$ go get -u google.golang.org/grpc@{{< param grpc_release_tag >}}
```

Comment:
Ok (I was having issues earlier -- it would pick up the package at head? -- but that was before I switched to using modules).

Reply:
Actually, I'll revert this to using the tag so that it will be easier to specify the path to the example source below. Also, some of the other quick start and tutorial pages refer to a specific release. (Release numbers are updated often enough :))